src/systems/filecoin_blockchain/storage_power_consensus/storage_power_actor_code.go

```diff
@@ -123,11 +58,8 @@ func (a *StoragePowerActorCode_I) WithdrawBalance(rt Runtime, amountRequested ac
 
 	UpdateRelease(rt, h, st)
 
-	// send funds to miner
-	rt.SendPropagatingErrors(&vmr.InvocInput_I{
-		To_:    ownerAddr,
-		Value_: amountExtracted,
-	})
+	// send funds to sender (pledge collateral)
+	rt.SendFunds(msgSender, amountExtracted)
```

Comment:
It's still preferable to leave this sending to the (uninitialized) owner address than sender.

Reply:
I think it makes sense for reward to always go to the owner. Pledge collateral is basically working capital, any reason why we would not want that to go the worker?

Reply:
Not sure we need to decide this now, but remitting to owner would be more consistent with SMA.